CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_zendesk_source v0.9.1
```

Comment:
I would prefer we make this a breaking change. Given the change in how the `group` source is quoted for Snowflake users, there is a small chance this could start failing out of the blue for some Snowflake users. As such, I would prefer we make this a breaking change in order to avoid this possibility. Let me know if you disagree.

Reply:
With this being a breaking change, be sure to make the proper updates elsewhere in the package.